src/git-info/index.ts

```diff
@@ -80,6 +80,7 @@ function addScriptsToPackageJson(): Rule {
 function addVersionGeneratorToGitignore(): Rule {
     return (tree: Tree, context: SchematicContext) => {
         addPropertyToGitignore(tree, context, 'src/environments/version.ts');
```

Comment:
Could you remove the old gitignore statement? Let's just go with one that always works instead of having 2